src/test/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannelTest.java

```diff
@@ -686,6 +686,23 @@ public void testInsertRowsWithSchemaEvolution() throws Exception {
 
       topicPartitionChannel.insertBufferedRecordsIfFlushTimeThresholdReached();
 
+      // Verify that the buffer is cleaned up and nothing is in DLQ because of schematization error
+      Assert.assertTrue(topicPartitionChannel.isPartitionBufferEmpty());
+      Assert.assertEquals(0, kafkaRecordErrorReporter.getReportedRecords().size());
+
+      // Do it again without any schematization error, and we should have row in DLQ
```

Comment:
i am confused about the comment, do you mean not have a row in DLQ if there is no schematization error?

or do you mean to verify the DLQ results something along this lines? Assert.assertEquals(0, kafkaRecordErrorReporter.getReportedRecords().size());?